app/app/Http/Controllers/SocialAuthController.php

```diff
@@ -42,6 +42,7 @@ private function createNewUserWithGoogle($profile) {
 			$newUser->email = $email;
 			$newUser->first_name = $profile->firstName;
 			$newUser->last_name = $profile->lastName;
+			$newUser->password_hash = User::generateSaltedHash('Hhaccess.2017');
```

Comment:
It looks like you could just remove commit 8b4a7a5e63adfc322c608be6536d6d7e65ca6a3a and that would remove the change quite well.  The commit message is "rebase the with the master and fix up default password to Hhaccess.2017"

You could remove with a 'git rebase -i master'. and removing the line 'pick ... rebase the with the master and fix up default password to Hhaccess.2017' before exiting vim.